packages/svelte2tsx/src/htmlxtojsx.ts

```diff
@@ -103,7 +97,11 @@ export function convertHtmlxToJsx(
     };
 
     const handleActionDirective = (attr: Node) => {
```

Comment:
`(attr: Node, parent: Node) =>`
I think we can pass the parent of the action as parameter it would be the element.

https://astexplorer.net/#/gist/4c18623d74b7906a0126287db9d78162/1e59c75a386605eaa9ca3c54fbbdbd3cbbb75aca